File: app/src/main/java/by/egorgutko/autorization/data/privatePreference/UserPreferencesInterface.java
package by.egorgutko.autorization.data.privatePreference;

import java.util.ArrayList;

import io.reactivex.Completable;
import io.reactivex.Single;

public interface UserPreferencesInterface {

    Single<ArrayList> getTaskList();
    Completable setUserTask(String task);
}

File: app/src/main/java/by/egorgutko/autorization/data/privatePreference/UserPreferencesInterface.kt
package by.egorgutko.autorization.data.privatePreference

import java.util.ArrayList

import io.reactivex.Completable
import io.reactivex.Single

interface UserPreferencesInterface {

    fun getTaskList(): Single<ArrayList<*>>
    fun setUserTask(task: String): Completable
}
